tailwind.config.ts

```diff
@@ -1,9 +1,9 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-var flattenColorPalette = require("tailwindcss/lib/util/flattenColorPalette").default;
+import flattenColorPalette from "tailwindcss/lib/util/flattenColorPalette";
```

Comment:
Awesome! I was hoping Tailwind would support TS someday. 🎉

Reply:
There are typings for tailwind, but they aren't there for v3 yet.